docs/cron-workflows.md

```diff
@@ -120,14 +121,14 @@ For example, if you want to stop scheduling new workflows after one success:
 
```

Comment:
Line 118 above needs to be changed now as well since you removed the `stopStrategy` section in the Fields Reference

Reply:
Good catch, fixed.